### PR TITLE
Classes

### PR DESCRIPTION
**Pull request by neyugntina**
add logo and classes page

### CODE REVIEW

File: src/_mock/products.js [hunk 1]
```diff
@@ -1,35 +1,26 @@
-import { faker } from '@faker-js/faker';
-import { sample } from 'lodash';
+import { faker } from "@faker-js/faker";
 
 // ----------------------------------------------------------------------
 
 const PRODUCT_NAME = [
```
Comment (n-grounds) [inline src/_mock/products.js:5]: change variable/const name from `PRODUCT_NAME` to something like `CLASSES`?

File: src/_mock/products.js [hunk 2]
```diff
@@ -39,19 +30,28 @@ const products = [...Array(24)].map((_, index) => {
...
-      (setIndex === 1 && PRODUCT_COLOR.slice(0, 2)) ||
-      (setIndex === 2 && PRODUCT_COLOR.slice(1, 3)) ||
-      (setIndex === 3 && PRODUCT_COLOR.slice(2, 4)) ||
-      (setIndex === 4 && PRODUCT_COLOR.slice(3, 6)) ||
-      (setIndex === 23 && PRODUCT_COLOR.slice(4, 6)) ||
-      (setIndex === 24 && PRODUCT_COLOR.slice(5, 6)) ||
-      PRODUCT_COLOR,
-    status: sample(['sale', 'new', '', '']),
+    title: PRODUCT_NAME[index + 1],
+    // price: faker.datatype.number({ min: 4, max: 99, precision: 0.01 }),
```
Comment (n-grounds) [inline src/_mock/products.js]: necessary to keep all this commented-out code?

File: src/components/homePage.component.js [hunk 3]
```diff
@@ -181,25 +172,25 @@ export default function MyApp({ mode, theme, colorMode }) {
                       // Use a switch statement or if-else to determine the route based on the icon
                       switch (index) {
```
Comment (n-grounds) [inline src/components/homePage.component.js:173]: seems like this `switch` statement is overkill, maybe something as simple as `navigate("/" + text.toLower());` would suffice?

File: src/_mock/products.js (hunk 1 above)
Comment (neyugntina) [reply src/_mock/products.js:5]: This variable is only temporary as this code will be overwritten when the API's are connected. Currently, this is only for mock data

**Review by rebekahre — APPROVED** [review]
gj tina

**Pull request merged**